src/utils/convert.util.ts

```diff
@@ -37,14 +45,23 @@ export function hex2rgb(hex: Color["hex"]): Color["rgb"] {
   const r = parseInt(hex.slice(0, 2), 16);
   const g = parseInt(hex.slice(2, 4), 16);
   const b = parseInt(hex.slice(4, 6), 16);
+  const a = parseInt(hex.slice(6, 8), 16);
+
+  if (!Number.isNaN(a)) {
+    return { r, g, b, a };
+  }
```

Comment:
Here the condition is not necessary, because our interface resolves `a` as `undefined`.

Reply:
This one I think we actually do need to keep, or at least something similar; `a` in this case is the result of calling `parseInt(hex.slice(6, 8), 16)`. `parseInt()` returns `NaN` if it cannot parse the value. So when calling this function with a six-digit hex string like `#FF007F`, the `parseInt()` will return `NaN` and you'll end up with `{ r: 255, g: 0, b: 127, a: NaN }` which is probably not what we want.